tests/test_dnatracing.py

```diff
@@ -0,0 +1,63 @@
+"""Tests of dnatracing methods."""
+import numpy as np
+import pytest
+
+
+def test_get_grain_array(test_dnatracing) -> None:
+    """Test extraction of grain array."""
+    TARGET = {
```

Comment:
Would this be better as a fixture? If only to future-proof?

Reply:
In `tests/test_dnatracing.py` I'm focusing on isolated testing of each method so there shouldn't be any need for each to be a `@pytest.fixture`.

However, I've setup a concurrent `tests/test_dnatracing_minicircle.py` which will take and test the `@pytest.fixture`'s defined in `tests/conftest.py` where there will be a "cascading" trail of fixtures, each built on the previous fixture. At the moment I've not started that cascade (although could have included one for `_get_grain_array()`. I'll make sure I add it in the next PR.